test-with-jest/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "test-with-jest",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "clear": "rm -rf dist && rm -rf coverage",
+    "start": "node dist/mock-function/index.js",
+    "test": "jest --detectOpenHandles dist --coverage",
+    "build": "npm run clear && tsc",
+    "lint": "eslint \"./src/**/*.ts\" --fix",
+    "format": "prettier \"./src/**/*.ts\" --write",
+    "precommit": "npm run format && git add . && npm run lint"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@azure/cosmos": "^4.1.0",
+    "@azure/identity": "^4.4.1",
+    "dotenv": "^16.4.5",
```

Comment:
I would recommend we try to minimize the number of 3p dependencies - they distract from the main message of this article (and are a source of upgrade pain, security issues, etc)

dotenv is the exception, although if you're targeting node there is support for --env-file https://nodejs.org/dist/latest-v22.x/docs/api/cli.html#--env-fileconfig 

if this is confusing, feel free to keep dotenv, but I recommend some simple hand-authored replacements for faker and uuid

Reply:
Keeping .env (this aligns with Cloud Advocacy samples and usage), replacing faker and UUID.